ci-operator/step-registry/hypershift/mce/ibmz/create/hcp/hypershift-mce-ibmz-create-hcp-commands.sh

```diff
@@ -102,8 +102,10 @@ data:
 EOF
 
 # Creating AgentServiceConfig
+CLUSTER_VERSION=$(oc get clusterversion -o jsonpath={..desired.version} | cut -d '.' -f 1,2)
+OS_IMAGES=$(jq --arg CLUSTER_VERSION "${CLUSTER_VERSION}" '[.[] | select(.openshift_version == $CLUSTER_VERSION)]' "${SHARED_DIR}/default_os_images.json")
 echo "$(date) Creating AgentServiceConfig"
-envsubst <<"EOF" | oc apply -f -
```

Comment:
I usually prefer `oc apply` over `oc create`, because both creates the resource but the apply overrides (when it's possible) the existent resource.

Reply:
done